MFRT/compile-all_windows.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Use GNU to get mfrt.o
+g++ -Wall -I<path to root include, example: /../root/include> -c mfrt.cpp
+
+# Use mfrt.o to link libraries and compile into exec format
+g++ -O2 -m64 mfrt.o -L</../root/lib> -lCore -lRIO -lNet -lHist -lGraf -lGraf3d -lGpad -lTree -lRint -lPostscript -lMatrix -lPhysics -lMathCore -lThread -pthread -lm -ldl -rdynamic -o mfrtROOT -L</../root/lib> -Wl,-R</../root/lib>
```

Comment:
We'll probably want to be careful about using absolute directories, unless we make something like a setup script to prepare the user's machine.

Not a problem for the prototype, just something we should think about.